modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -413,10 +569,10 @@ def _apply_list_of_funcs(call_queue, partition):  # pragma: no cover
     str
         The node IP address of the worker process.
     """
-    for func, f_args, f_kwargs in call_queue:
-        func = deserialize(func)
-        args = deserialize(f_args)
-        kwargs = deserialize(f_kwargs)
+    call_queue = reconstruct_call_queue(
```

Comment:
@dchigarev should we do the same for https://github.com/modin-project/modin/blob/b608ea056ef164da4730a5db5b0ac9a82e6e803c/modin/core/execution/ray/implementations/pandas_on_ray/partitioning/virtual_partition.py#L313?

Reply:
done